### PR TITLE
api: update composer recipe hautelook/alice-bundle to 2.13

### PR DESCRIPTION
**Pull request by BacLuc**
Optimization of putting the 2 config files for dev and test in one file and sharing the config was not needed, because we only use it for test.

### CODE REVIEW

**Review by carlobeltrame — APPROVED** [review]
Change lgtm. Not sure what config files you are talking about.

**Comment by BacLuc** [comment]
> Change lgtm. Not sure what config files you are talking about.

It's about this file: https://github.com/ecamp/ecamp3/blob/42fbf82a36f8fd7c625f4bf95409d5a52d6c9532/api/config/packages/test/hautelook_alice.yaml

File: api/symfony.lock [hunk 1]
```diff
@@ -193,16 +193,15 @@
...
-            "version": "2.1",
-            "ref": "71822522faf7ed2792d86b7f94ce73443358ccb9"
+            "branch": "main",
+            "version": "2.2",
+            "ref": "c84e4f2b9d7f436d7d52e8369230b393367607ec"
         },
         "files": [
-            "config/packages/dev/hautelook_alice.yaml",
-            "config/packages/test/hautelook_alice.yaml",
+            "config/packages/hautelook_alice.yaml",
```
Comment (pmattmann) [inline api/symfony.lock:204]: I am also confused. This file does not exist in the source.

**Comment by BacLuc** [comment]
not important right now

**Pull request closed without merging**